src/reporter.test.ts

```diff
@@ -311,13 +311,124 @@ describe('reporter', () => {
             httpInfo: {
               host: 'your.mind.com',
               request: {
+                truncated: false,
                 body: '{"secret":"****"}',
                 headers: '{"Tyler":"Durden","secretKey":"****","content-type":"application/json"}',
                 host: 'your.mind.com',
               },
               response: {
                 truncated: true,
-                body: `{"a":"${manyA}","b":"${manyA}","key":"****","password":"****","e":"${manyA}","secret":"****","f":"${manyA}","g":"${manyA}","h":"${manyManyA}"}`,
+                body: `{"a":"${value}","b":"${value}","key":"****","password":"****","e":"${value}","secret":"****","f":"${value}","g":"${value}"}`,
+                headers: '{"Peter":"Parker","content-type":"application/json"}',
+              },
+            },
+          },
+        },
+        { dummyEnd },
+      ];
+      const expectedResultSize = getJSONBase64Size(spans);
+
+      const actual = JSON.parse(reporter.forgeAndScrubRequestBody(spans, expectedResultSize));
+      expect(actual).toEqual(expected);
+    });
+    test('forgeAndScrubRequestBody long response', () => {
+      const value = 'a'.repeat(10);
+      const long = 'a'.repeat(getEventEntitySize(true));
+      const shorter = 'a'.repeat(getEventEntitySize());
+      const dummyEnd = 'dummyEnd';
+      const spans = [
+        {
+          info: {
+            httpInfo: {
+              host: 'your.mind.com',
+              request: {
+                truncated: false,
+                host: 'your.mind.com',
+                headers: {
+                  Tyler: 'Durden',
+                  secretKey: 'lumigo',
+                  'content-type': 'application/json',
+                },
+                body: '{"secret": "secret"}',
+              },
+              response: {
+                truncated: false,
+                headers: { Peter: 'Parker', 'content-type': 'application/json' },
+                body: `{"key":"${value}","password":"${value}","e":"${value}","secret":"${value}","f":"${value}","g":"${value}","h":"${long}"}`,
+              },
+            },
+          },
+        },
+        { dummyEnd },
+      ];
+      const expected = [
+        {
+          info: {
+            httpInfo: {
+              host: 'your.mind.com',
+              request: {
+                truncated: false,
+                body: '{"secret":"****"}',
+                headers: '{"Tyler":"Durden","secretKey":"****","content-type":"application/json"}',
+                host: 'your.mind.com',
+              },
+              response: {
+                truncated: false,
+                body: `{"key":"****","password":"****","e":"${value}","secret":"****","f":"${value}","g":"${value}","h":"${shorter}"}...[too long]`,
+                headers: '{"Peter":"Parker","content-type":"application/json"}',
```

Comment:
This one looks like it's truncated, because of the `...[to long]`

Reply:
truncated not related to [too long] it indicates either we stoped collecting the body